gr-qtgui/python/qtgui/compass.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+#
+
+import numpy
+from gnuradio import gr;
+import pmt
+import time
+
+#from PyQt5.QtWidgets import QApplication, QWidget,  QVBoxLayout, QHBoxLayout, QGridLayout,  QSpinBox,  QSlider
+#from PyQt5.QtCore import QSize, pyqtSignal,  QPoint,  pyqtSlot,  pyqtProperty,  Qt
+#from PyQt5.QtGui import QPainter, QPalette,  QFont,  QFontMetricsF,  QPen,  QPolygon
+
+# First Qt and 2nd Qt are different.  You'll get errors if they're both not available, hence the import-as to avoid name collisions
+from PyQt5 import Qt, QtGui
+from PyQt5.QtCore import Qt as Qtc
+from PyQt5.QtCore import pyqtSignal,  QPoint,  pyqtSlot,  pyqtProperty, QRect
```

Comment:
some double spaces here and in the imports lines below.

pylint will catch these